test/parallel/test-error-codes-impl-doc.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+// Flags: --expose-internals
+
+// Note: this script reads Node.js sources and docs, and expects to be run on
+// the very same Node.js version as the source and doc tree is.
+
+require('../common');
+const assert = require('assert');
+const errors = require('internal/errors').codes;
+const fs = require('fs');
+const path = require('path');
+const natives = process.binding('natives');
+
+// --report prints only the list of failed checks and does not assert
+const report = process.argv[2] === '--report';
+const results = [];
+function check(ok, type, name, reason) {
+  const label = `${type}: ${name} - ${reason}`;
+  if (report) {
+    if (!ok) results.push(label);
+  } else {
+    assert.ok(ok, label);
+  }
+}
+
+const set = {
+  all: new Set(),
+  js: new Set(), // found in natives
+  doc: new Set(), // mentioned in doc/api/*.md
+  cpp: new Set(), // found in src/node_errors.h
+  noargs: new Set(), // subset of `js` which is constructed without arguments
+  // `set.manual` contains errors manually created from js and not defined in
```

Comment:
It is not immediately clear if this comment is a continuation of the previous line or a new section. Maybe uppercase the previous (if not all) line and add a period or insert a line break before?